daemon/src/projection.rs

```diff
@@ -1099,22 +1099,22 @@ impl State {
                     contract_symbol: ContractSymbol::BtcUsd,
                     position_maker: Position::Long,
                     ..
-                } => self.offers.btcusd_long = Some(new_offer),
+                } => self.offers.btcusd_short = Some(new_offer),
```

Comment:
🔧 I think an explanatory comment would be helpful to explain why we switch here from long to short (and vice versa), as you mentioned it's not obvious from just looking at the code..